legal-api/devops/vaults.gcp.env

```diff
@@ -14,7 +14,7 @@ GUNICORN_THREADS=8
 GUNICORN_TIMEOUT=0
 
 # Launchdarkly
-LD_SDK_KEY="op://launchdarkly/$APP_ENV/business-filings/BUSINESS_FILING_LD_CLIENT_ID"
+LD_SDK_KEY="op://launchdarkly/$APP_ENV/business-api/BUSINESS_API_LD_SDK_KEY"
```

Comment:
This is `legal-api` branch. Should it be Business-API specific?

Reply:
> I'm still curious how this worked before with the wrong key.

I think it's because OCP was using secrets, but GCP is now using 1Password. This makes sense to me 😄